src/NuGet.Clients/PackageManagement.VisualStudio/NuGet.PackageManagement.VisualStudio.csproj

```diff
@@ -88,13 +88,20 @@
   <ItemGroup>
     <Compile Include="BindingRedirectBehavior.cs" />
     <Compile Include="CommonResources.cs" />
-    <Compile Include="IDE\NuGetAndEnvDTEProjectCache.cs" />
-    <Compile Include="IDE\EnvDTEProjectName.cs" />
+    <Compile Include="IDE\IVsSolutionManager.cs" />
+    <Compile Include="ProjectSystems\CpsPackageReferenceProject.cs" />
```

Comment:
@rohit21agrawal Agree. As this PR changes build fine for both VS14/VS15 toolsets I'd suggest to handle this in a separate PR when it's clear what parts should be conditionally included.